words/words.md

```diff
@@ -686,10 +686,10 @@ Isolare i casi *sintomatici* ridurrebbe R fino al 40%, e mettere i loro contatti
     * Asymptomatics contribute R = 0.1 (5%, though their model has uncertainty and it could be much lower)
     * Environmental stuff like doorknobs contribute R = 0.2 (10%)
 -->
-    * I casi sintomatici contribuiscono a R = 0.8 (40%)
-    * I casi presintomatici contribuiscono a R = 0.9 (45%)
-    * I casi asintomatici contribuiscono a R = 0.1 (5%, though their model has uncertainty and it could be much lower)
-    * Gli elementi dell'ambiente, come le maniglie delle porte, contribuiscono a R = 0.2 (10%)
+    * I casi sintomatici contribuiscono ad R per lo 0.8 (40%)
```

Comment:
regarding this and possibly other numerical values, I would change all "0.x" to "0,x" (I think I mostly did in the parts I translted) as we need to use "," in Italian. You could do it in this PR or we can do it afterwards).

Reply:
No, I do not think we should do that. I think when it comes to technical writing people are used to see dots. I'll leave it for another pull request in any case.